gcsfs/core.py

```diff
@@ -124,24 +124,30 @@ class GCSFileSystem(object):
     """
     Connect to Google Cloud Storage.
 
-    Two modes of authentication are supported:
-    - if ``token=None``, GCSFS will attempt to use your default google
-      credentials; if this is not found, you will be given a "device code",
-      which you must enter into a browser where you are logged in with your
-      Google identity.
+    The following modes of authentication are supported:
+    - ``token=None``, GCSFS will attempt to guess your credentials in the
+      following order: gcloud CLI default, gcsfs cached token, google compute
+      metadata service, anonymous.
+    - ``token='gtoken'``, your default gcloud credentials will be used, which
+      are typically established by doing ``gcloud login`` in a terminal.
```

Comment:
Is `gtoken` a standard term?

Reply:
No, but I wanted something different from the previously provided "cloud" term. "gcloud" might otherwise have been more reasonable.